### PR TITLE
Complete my portfolio

### PR DESCRIPTION
**Pull request by fanhualta**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,53 @@
...
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 style="text-align:center">This is Tianan Li's portfolio</h1>
+      <hr>
+      <h2>Who I am</h2>
+      <div>
+      <div class="photo"><img style="max-width:80%;overflow:hidden;text-align:center;" src="images/photo.jpg"></div>
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/index.html]: An overall comment for this whole file about formatting: could you indent the html sub-elements?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -4,6 +4,32 @@
   width: 650px;
 }
 
-#greeting-container {
+#container {
   margin-top: 20px;
 }
+
+p, li{
+    word-break: keep-all;
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/style.css:12]: Please keep the indentation consistent: use either 2 spaces or 4 spaces for all appearances.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -4,6 +4,32 @@
...
+    word-wrap: break-word;
+    white-space: pre-wrap;
+    text-align:justify;
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/style.css]: Please add a space before.

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -4,6 +4,32 @@
...
+    float:right;
+    width:40%;
+}
+
+.photo{
+    float:left;
+    width:25%;
+}
+
+
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/style.css]: Delete redundant empty lines?

**Review by yuxifeng-sps — COMMENTED** [review]
Overall looks fine, just some nits.

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (fanhualta) [reply portfolio/src/main/webapp/style.css]: Fixed

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (fanhualta) [reply portfolio/src/main/webapp/style.css]: Fixed

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (fanhualta) [reply portfolio/src/main/webapp/style.css:12]: Fixed

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (fanhualta) [reply portfolio/src/main/webapp/index.html]: Thanks for your suggestions. Fixed.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -26,7 +35,60 @@
...
+  }
+
+
+  private final class Comment {
+
+    private final String comment;
+    private final long timestamp;
+
+    public Comment(String comment, long timestamp) {
+        this.comment = comment;
```
Comment (yuxifeng-sps) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Maybe just need 2 whitespaces here, to be consistent with other indentations.

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -2,17 +2,69 @@
...
+      <h2>Who I am</h2>
+      <div>
+        <div class="photo"><img style="max-width:80%;overflow:hidden;text-align:center;" src="images/photo.jpg"></div>
+        <div><p>My name is Tianan Li, from the School of Software of Tsinghua University. I am currently in the second year of a graduate student. My major is software engineering, and my research direction is database kernel R&D. Undergraduate studies in communications engineering at Beijing University of Posts and Telecommunications. My native place is Anhui. I grew up in Tianjin since I was a child, so I'm named Tianan. My name has nothing to do with Tian'anmen Square, it is just a coincidence.</p></div>
+      </div>
+      <hr>
+      <h2><strong>Here's what I've done so far</strong></h2>
+      <div>
+        <div class="title">
+            <a href="https://iotdb.apache.org/"><h1 style="text-align:center"><strong>Apache IoTDB</strong></h1></a>
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/index.html]: Ditto.

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -2,17 +2,69 @@
...
+        </div>
+	    <div class="logo">
+            <img style="max-width:80%;overflow:hidden;" src="images/logo.png">
+        </div>
+	    <h3 class="intro">Introduction</h3>
+        <p>Apache IoTDB is the only Apache project in China’s universities. It is a timeseries database for industrial Internet independently developed by Tsinghua University. IoTDB(Internet of Things Database) is an integrated data management engine designed for timeseries data, which can provide users specific services for data collection, storage and analysis. Due to its light weight structure, high performance and usable features together with its intense integration with Hadoop and Spark ecology, IoTDB meets the requirements of massive dataset storage, high-speed data input and complex data analysis in the IoT industrial field.</p>
+        <h3>Main Work</h3>
+        <ul>
+            <li>One of Apache IoTDB founding team core members and PPMC members, responsible for the kernel development. Independently developed the synchronization module, dynamic parameter adjustment module, memory estimation tool, etc; Participate in refactoring the write engine and building the Apache IoTDB website.</li>
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/index.html]: Ditto.

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -2,17 +2,69 @@
...
+            <li>The head of the Apace IoTDB distributed version development team. Designed and implemented a distributed timeseries database framework based on Raft protocol and consistent hashing. Designed and implemented distributed metadata management, writing, and query modules. Contributed more than 50% of the codes.</li>
+            <li>Maintain the open-source community ecology, introduce Apache IoTDB and discuss the deployment of Apache IoTDB with multiple enterprises.</li>
+        </ul>
+      </div>
+      <hr>
+      <div>
+        <h2><strong>Learn together</strong></h2> 
+        <ul>
+            <li><a href="https://github.com/fanhualta"><h3><strong>My repo</strong></h1></a></li>
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/index.html]: Ditto.

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -2,17 +2,69 @@
...
+      <div>
+        <h2><strong> My Motto </strong></h2>
+        <p> Never Give Up! </p>
+      </div>
+      <div>
+        <h2><strong> All comments </strong></h2>
+        <ul id="show-comments-container"></ul>
+      </div>
+      <div>
+          <form action="/data" method="POST">
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/index.html]: Ditto.

File: portfolio/src/main/webapp/script.js [hunk 10]
```diff
@@ -15,14 +15,35 @@
...
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
+  const greetingContainer = document.getElementById('container');
   greetingContainer.innerText = greeting;
 }
+
+function showComments() {
+    fetch('/data').then(response => response.json()).then((stats) => {
```
Comment (yuxifeng-sps) [inline portfolio/src/main/webapp/script.js]: Ditto.

**Review by yuxifeng-sps — COMMENTED** [review]
Overall looks good, again just some nits.

**Review by yuxifeng-sps — APPROVED** [review]

**Pull request merged**